app/views/boards/_reorder_posts.haml

```diff
@@ -1,3 +1,6 @@
+-# locals: ( unsectioned_posts:, id: nil, klass: '' )
+-# TODO: klass is not used but is passed from indexes#edit
```

Comment:
Looks like `klass` is only passed in by `indexes/edit.haml` (like you said), but it's IndexPost instead of a string / the usual cycle logic. I think it's a mistake, we can get rid of it.